definitions/ext-dora/summary_metrics.yml

```diff
@@ -0,0 +1,25 @@
+doraName:
+  tag:
+    key: doraName
+  title: DORA
+  unit: STRING
+deploymentFrequency:
+  tag:
+    key: deploymentFrequency
+  title: Deployment Frequency
+  unit: COUNT
+leadTimeForChanges:
+  tag:
```

Comment:
These should point to the golden metrics created.

```
leadTimeForChanges:
  goldenMetric: leadTimeForChanges
  title: Lead Time For Changes
  unit: COUNT
```

Reply:
updated 👍🏾